tools/castle-editor/code/editorutils.pas

```diff
@@ -144,12 +150,21 @@ procedure SetEnabledExists(const C: TControl; const Value: Boolean);
 function ApiReference(const SelectedObject: TObject;
   const PropertyName, PropertyNameForLink: String): String;
 
+procedure BuildComponentsMenu(const ParentUeserInterface, ParentTransform: TMenuItem; const OnClickEvent: TNotifyEvent);
+
 implementation
 
 uses SysUtils, Dialogs, Graphics, TypInfo,
   CastleUtils, CastleLog,
+  CastleComponentSerialize, CastleUiControls, CastleCameras, CastleTransform,
   ToolCompilerInfo;
 
+procedure TMenuItemHelper.SetEnabledVisible(const Value: Boolean);
+begin
+  Visible := Value;
+  Self.Enabled := Value;
```

Comment:
Write just `Enabled`, not `Self.Enabled`. It seems inconsistent you use `Visible` but `Self.Enabled` :)

Reply:
Yes, I wanted to get some completion hint, but then forgotten to delete the Self prefix. Fixed in 9de9cd08c6886184e18f5ca81d866b4b8be0ed0e